users/config.rst

```diff
@@ -490,9 +546,6 @@ skipIntroductionRemovals
 introducedBy
     Defines which device has introduced us to this device. Used only for following de-introductions.
 
-certName
-    The device certificate common name, if it is not the default "syncthing".
-
```

Comment:
Is there a specific reason to move this further up?  There is no alphabetical sorting, dunno what order the JSON serialization applies otherwise.